src/db.js

```
@@ -0,0 +1,19 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+const utils = require('util');
+
+require('dotenv').config();
+global.TextEncoder = utils.TextEncoder;
+
+const { DB_HOST, DB_PORT, DB_USER, DB_PASSWORD, DB_NAME } = process.env;
+const sequelize = new Sequelize({
+  database: DB_NAME || 'postgres',
+  username: DB_USER || 'postgres',
+  host: DB_HOST || 'localhost',
+  dialect: 'postgres',
+  port: DB_PORT || 5432,
+  password: DB_PASSWORD || '123',
```

Avoid using insecure or permissive default credentials (`password: '123'`) because they silently mask missing environment configuration and can lead to accidental usage of unsafe defaults. Either require these env vars and throw an error if missing, or use `null`/`undefined` and fail fast. At minimum log a clear error when essential DB env vars are absent.